tools/pot/openvino/tools/pot/data_loaders/creator.py

```diff
@@ -22,11 +23,19 @@ def create_data_loader(config, model):
                            'Actual inputs number: {}'.format(len(inputs)))
 
     data_loader = None
+    assert len(inputs) == 1 or config.type == 'simplified'
```

Comment:
Maybe it would be better to expand the conditions above? With RuntimeError and human-readable message?

Reply:
done